src/amo/reducers/users.js

```diff
@@ -5,10 +5,12 @@ import {
   ADDONS_CONTENTREVIEW,
   ADDONS_EDIT,
   ADDONS_POSTREVIEW,
+  ADDONS_RECOMMENDED_REVIEW,
```

Comment:
`ADDONS_RECOMMENDREVIEW` to be consistent with the underlying permission string?

Reply:
@eviljeff I was going to change the existing permission names to match addons-server names... What do you think? I decided not to do it here to keep the patch small but I can definitely do it here.

Reply:
I don't have any strong opinions.  Either consistency with either the addons-server permission constant names or the strings.